docs/ops.html

```diff
@@ -2988,7 +2988,7 @@ <h5 class="anchor-heading"><a id="kafka_streams_thread_monitoring" class="anchor
       </tr>
       <tr>
         <td>blocked-time-ns-total</td>
-        <td>The total time the thread spent blocked on kafka.</td>
+        <td>The total time in ns the thread spent blocked on kafka.</td>
```

Comment:
nit:
```suggestion
        <td>The total time in nanoseconds the thread spent blocked on kafka.</td>
```

Reply:
Well, we use `ns / ms` everywhere... If we update it here, we must update basically _all_ metrics. I tired to make a minimum change. Let me know what you think.

Reply:
I feel comfortable with `ns/ms`. Honestly either is fine, but if we want to keep the changes small I'd say just go for this :)

Reply:
Talked to Bruno offline, and he is also fine with the abbreviations.

I took one more look and also changed `seconds` to `sec` to use an abbreviation also there for consistency, and made a few other changes/fixed to get a unified way how docs are written.

If there is no objections raised, I will merge this in the next couple of days.

Reply:
No objections.